### DISCUSSION TITLE
Feature Question - Path Signals + PAX destinations?

### DISCUSSION THREAD

**Discussion by xRaynex** [OpenLoco/OpenLoco#1272] [Q&A] [answered]
Hi there guys! I know you're working on the backend and that feature updates are not the priority now. I just wanted to ask if these two things are considered to be somewhere on the roadmap. Path signals I figure is a pretty integral part of this sort of project regardless, but I know individual passenger destinations can be a little less of a priority. It'd be great to know if they stand a chance at coming sometime in the (relatively) near/far/whereveryouare future.

**Comment by AaronVanGeffen** [comment, marked as answer]
It will probably happen years down the line. But yes, eventually! It's already been requested and tracked it in #80 🙂
